### PR TITLE
chore: inital 13.0.0 blog post

### PR DESCRIPTION
**Pull request by VerteDinde**
cc @sofianguy @clavin @georgexu99

Please add a comment for items you think we should highlight in the blog post!

⚠️ do not merge until the following are completed ⚠️ 

- [x]  update node, v8 and chromium versions from final chrome roll under Stack Changes section
- [x]  add a couple new features in tag line sentence
- [x]  edit link for M91 blog post
- [x]  add a few bullets for Highlight Features section
- [x]  add any missing items in Breaking Changes section
- [x]  add items for API Changes section
- [x]  add items for Deprecated APIs section

### CODE REVIEW

File: data/blog/electron-13-0.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Electron 13.0.0
+author:
+- VerteDinde
+- clavin
+- georgexu99
+date: '2021-05-25'
+---
+
+Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `x.x` and Node.js `14.x`. We've added {highlighted changes}, and general improvements. Read below for more details!
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:10]:
```suggestion
Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `9.1` and Node.js `14.16.0`. We've added {highlighted changes}, and general improvements. Read below for more details!
```

File: data/blog/electron-13-0.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+The Electron team is excited to announce the release of Electron 13.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `91`
+    * [New in Chrome 88](https://developer.chrome.com/blog/new-in-chrome-88/)
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:21]:
```suggestion
    * [New in Chrome 91](https://developer.chrome.com/blog/new-in-chrome-91/)
```

File: data/blog/electron-13-0.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    * [New in Chrome 89](https://developer.chrome.com/blog/new-in-chrome-89/)
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:22]:
```suggestion
    * [New in Chrome 90](https://developer.chrome.com/blog/new-in-chrome-90/)
```

File: data/blog/electron-13-0.md (hunk 2 above)
Comment (sofianguy) [reply data/blog/electron-13-0.md:21]: The new in chrome 91 link is not live yet, but pretty sure they're following this URL convention

File: data/blog/electron-13-0.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* Node.js `14.x`
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:23]:
```suggestion
* Node.js `14.16.0`
```

File: data/blog/electron-13-0.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    * [Node 14.16.0 blog post](https://nodejs.org/en/blog/release/v14.16.0/)
+    * [Node 14.0.0 blog post](https://nodejs.org/en/blog/release/v14.0.0/)
+* V8 `x.x`
+    * [V8 8.8 blog post](https://v8.dev/blog/v8-release-88)
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:27]:
```suggestion
    * [V8 9.1 blog post](https://v8.dev/blog/v8-release-91)
```

File: data/blog/electron-13-0.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    * [V8 8.9 blog post](https://v8.dev/blog/v8-release-89)
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:28]:
```suggestion
    * [V8 9.0 blog post](https://v8.dev/blog/v8-release-90)
```

**Review by sofianguy — CHANGES_REQUESTED** [review]
Thanks for getting this started, @VerteDinde !

File: data/blog/electron-13-0.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+## API Changes
+
+* Added `roundedCorners` option for `BrowserWindow`. [#27572](https://github.com/electron/electron/pull/27572)
+* Added new `session.storagePath` API to get the path on disk for session-specific data.[28866](https://github.com/electron/electron/pull/28866)
+* Added support for passing DOM elements over the context bridge. [#26776](https://github.com/electron/electron/pull/26776)
+* Added `process.uptime()` to sandboxed renderers. [#26684](https://github.com/electron/electron/pull/26684)
+* Added missing fields to the parameters emitted as part of the `context-menu `event.[#26788](https://github.com/electron/electron/pull/26788)
+* Added support for registering Manifest V3 extension service workers.
+Added ‘registration-completed’ event to ServiceWorkers. [#27562](https://github.com/electron/electron/pull/27562)
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:53]:
```suggestion
* Added ‘registration-completed’ event to ServiceWorkers. [#27562](https://github.com/electron/electron/pull/27562)

```

File: data/blog/electron-13-0.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+### Stack Changes
+
+* Chromium `91`
+    * [New in Chrome 91](https://developer.chrome.com/blog/new-in-chrome-91/)
+    * [New in Chrome 90](https://developer.chrome.com/blog/new-in-chrome-90/)
+* Node.js `14.16.0`
+    * [Node 14.16.0 blog post](https://nodejs.org/en/blog/release/v14.16.0/)
+    * [Node 14.0.0 blog post](https://nodejs.org/en/blog/release/v14.0.0/)
+* V8 `x.x`
```
Comment (sofianguy) [inline data/blog/electron-13-0.md:26]:
```suggestion
* V8 `9.1`
```

**Review by sofianguy — CHANGES_REQUESTED** [review]

**Review by sofianguy — APPROVED** [review]
🚀

File: data/blog/electron-13-0.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+* V8 `9.1`
+    * [V8 9.1 blog post](https://v8.dev/blog/v8-release-91)
+    * [V8 9.0 blog post](https://v8.dev/blog/v8-release-90)
+
+### Highlight Features
+
+* Added `process.contextIsolated` property that indicates whether the current renderer context has `contextIsolation` enabled. [#28252](https://github.com/electron/electron/pull/28252) 
+* Added new `session.storagePath` API to get the path on disk for session-specific data. [#28866](https://github.com/electron/electron/pull/28866) 
+* Deprecated the `new-window` event of `WebContents`. It is replaced by `webContents.setWindowOpenHandler()`
+* The `submitURL` option for `crashReporter.start` is no longer a required argument when `uploadToServer` is false. [#28283](https://github.com/electron/electron/pull/28283)
```
Comment (MarshallOfSound) [inline data/blog/electron-13-0.md:35]: This one feels like more of a bugfix and not up at the same level as the other items in this section?

File: data/blog/electron-13-0.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+date: '2021-05-25'
+---
+
+Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `9.1` and Node.js `14.16.0`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 13.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
```
Comment (MarshallOfSound) [inline data/blog/electron-13-0.md:16]: I know we haven't fully figured it out yet (at least the electron-rebuild part) but do we want to mention the native module thing here? Depends on whether someone can figure out the electron-rebuild thing today I guess.

Comment (VerteDinde) [reply data/blog/electron-13-0.md:16]: Working on getting the electron-rebuild part working, I'll add the native module fix here, that's a good callout 🙂

File: data/blog/electron-13-0.md (hunk 9 above)
Comment (VerteDinde) [reply data/blog/electron-13-0.md:35]: Yeah, true - I'll remove this one

File: data/blog/electron-13-0.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+* Removed the following deprecated `BrowserWindow` extension APIs:
+
+    * `BrowserWindow.addExtension(path)`
+    * `BrowserWindow.addDevToolsExtension(path)`
+    * `BrowserWindow.removeExtension(name)`
+    * `BrowserWindow.removeDevToolsExtension(name)`
+    * `BrowserWindow.getExtensions()`
+    * `BrowserWindow.getDevToolsExtensions()`
+
+    Use the session APIs instead:
```
Comment (erickzhao) [inline data/blog/electron-13-0.md:71]:
```suggestion
    Use the `session` APIs instead:
```

File: data/blog/electron-13-0.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+    * `ses.removeExtension(extension_id)`
+    * `ses.getAllExtensions()`
+
+* The following systemPreferences methods have been deprecated:
+
+    * `systemPreferences.isDarkMode()`
+    * `systemPreferences.isInvertedColorScheme()`
+    * `systemPreferences.isHighContrastColorScheme()`
+
+    Use the following nativeTheme properties instead:
```
Comment (erickzhao) [inline data/blog/electron-13-0.md:83]:
```suggestion
    Use the following `nativeTheme` properties instead:
```

File: data/blog/electron-13-0.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+    * `BrowserWindow.getExtensions()`
+    * `BrowserWindow.getDevToolsExtensions()`
+
+    Use the session APIs instead:
+
+    * `ses.loadExtension(path)`
+    * `ses.removeExtension(extension_id)`
+    * `ses.getAllExtensions()`
+
+* The following systemPreferences methods have been deprecated:
```
Comment (erickzhao) [inline data/blog/electron-13-0.md:77]:
```suggestion
* The following `systemPreferences` methods have been deprecated:
```

File: data/blog/electron-13-0.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
+---
+title: Electron 13.0.0
+author:
+- sofianguy
+- georgexu99
+- VerteDinde
+date: '2021-05-25'
+---
+
+Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `9.1` and Node.js `14.16.0`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
```
Comment (erickzhao) [inline data/blog/electron-13-0.md:10]: We're on the same Node.js version right? 
```suggestion
Electron 13.0.0 has been released! It includes upgrades to Chromium `91` and V8 `9.1`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
```

File: data/blog/electron-13-0.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+The Electron team is excited to announce the release of Electron 13.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `91`
+    * [New in Chrome 91](https://www.youtube.com/watch?v=vy6FXa0n1r0)
+    * [New in Chrome 90](https://developer.chrome.com/blog/new-in-chrome-90/)
+* Node.js `14.16.0`
```
Comment (nomagick) [inline data/blog/electron-13-0.md:23]: Actually 14.17.0

Comment (VerteDinde) [reply data/blog/electron-13-0.md:23]: It's still 14.16.0 🙂

File: data/blog/electron-13-0.md (hunk 10 above)
Comment (VerteDinde) [reply data/blog/electron-13-0.md:16]: Still getting this up here: https://github.com/electron/electron-rebuild/pull/703 I'll leave out of the blog post for now and add in when we add the 91 blog post.

**Review by erickzhao — APPROVED** [review]

**Pull request merged**